portal/get_patient_documents.php

```diff
@@ -1,160 +1,131 @@
 <?php
 
 /**
- * Download documents from OpenEMR to the patient portal in a zip file(get_patient_documents.php)
- *
- * This program is used to download patient documents in a zip file in the Patient Portal.
- * The original author did not pursue this but I thought it would be a good addition to
- * the patient portal
- *
- * @package   OpenEMR
- * @link      https://www.open-emr.org
- * @author    Giorgos Vasilakos <giorg.vasilakos@gmail.com>
- * @author    Terry Hill <terry@lilysystems.com>
- * @author    Stephen Waite <stephen.waite@cmsvt.com>
- * @copyright Copyright (c) 2012 Giorgos Vasilakos <giorg.vasilakos@gmail.com>
- * @copyright Copyright (c) 2015-2017 Terry Hill <terry@lillysystems.com>
- * @copyright Copyright (c) 2019 Stephen Waite <stephen.waite@cmsvt.com>
- * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * Display and download patient documents with checkboxes
  */
 
-    require_once("verify_session.php");
-    require_once("$srcdir/documents.php");
-    require_once($GLOBALS['fileroot'] . "/controllers/C_Document.class.php");
+require_once("./verify_session.php");
+$ignoreAuth_onsite_portal = true;
+require_once('../interface/globals.php');
```

Comment:
should remove above 2 lines since verify_session.php already does this (potential security issue)